spec/big_files/big_files_spec.rb

```diff
@@ -14,14 +14,18 @@
   end
 
   [{ glob: nil, exclude: nil },
-   { glob: '*/*.{rb,swift}', exclude: '*/foo.rb' }].each do |config|
+   { glob: '*/*.{rb,swift}', exclude: '*/foo.rb' },
+   { glob: '*/*.{rb,swift}', exclude: '*/foo.rb',
+     num_files: '6' }].each do |config|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.